db/seeds.rb

```diff
@@ -5,3 +5,8 @@
/sites/default/files/WARP230_Packshot_480.jpg", release_year: "2012", price: 600, genre: Genre.second, artist: Artist.fourth, slug: "until-the-quiet-comes" }, { title: "Cosmogramma", description: "Third studio album by producer Flying Lotus. Features collaborations with Thundercat and Thom Yorke.", image_url: "https://warpnet-media.s3.amazonaws.com/c87133e3d521e85d320f7f5b2241cd3bf658cd255f1d87c315fa40fd", release_year: "2010", price: 500, genre: Genre.second, artist: Artist.fourth, slug: "cosmogramma" }, { title: "Luxury Problems", description: "Andy Stott's third studio release, featuring work from the Homeless Gospel Choir", image_url: "http://www.tinymixtapes.com/sites/default/files/1210/andy_stott-luxury_problems.jpg", release_year: "2012", price: 600, genre: Genre.second, artist: Artist.fifth, slug: "luxury-problems" }, { title: "Illmatic", description: "The greatest hip-hop album of all time. Period.", image_url: "https://www.billboard.com/files/media/nas-illmatic-billboard-510.jpg", release_year: "1994", price: 900, genre_id: Genre.third, artist: Artist.find(6), slug: "illmatic" }, { title: "untitled unmastered.", description: "Kendrick Lamar responds to his Album of the Year snub with leftover tracks from 'To Pimp a Butterfly'.", image_url: "http://static.stereogum.com/uploads/2016/03/kendrick-lamar-untitled-unmastered-surprise-new-album-compressed1-compressed.jpg", release_year: "2016", price: 5000, genre: Genre.third, artist: Artist.find(7), slug: "untitled-unmastered" }, { title: "To Pimp A Butterfly", description: "Artistically-driven, jazzy follow-up to his debut, Kendrick teams up with the likes of George Clinton, Thundercat, Kamasi Washington, and Snoop Lion to deliver arguably the greatest album of 2015.", image_url: "http://media.cleveland.com/ent_impact_home/photo/pimp-a-butterfly-aa71fd87e2c23639.png", release_year: "2015", price: 900, genre: Genre.third, artist: Artist.find(7), slug: "to-pimp-a-butterfly" }, { title: "College Dropout", description: "Recorded over four years, Yeezy's debut album marked his place as a top solo artist in the industry.", image_url: "http://images.complex.com/complex/image/upload/t_article_image/kap7sz3mbnrnrhdugnjl.jpg", release_year: "2004", price: 700, genre: Genre.third, artist: Artist.find(8), slug: "college-dropout" }, { title: "The Life of Pablo", description: "Buy it before the cease and decist letter arrives.", image_url: "https://timedotcom.files.wordpress.com/2016/02/kanye-west-the-life-of-pablo.jpg?quality=75&strip=color&w=650", release_year: "2016", price: 10000, genre: Genre.third, artist: Artist.find(8), slug: "life-of-pablo" }, { title: "Low End Theory", description: "Often credited with defining the sound of 90's hip-hop, 'Low End Theory' sees TCQ ascend to their peak in rhyme and beats.", image_url: "https://images.rapgenius.com/aa2ffacda362f9578ff8c3f4f32f11fc.500x500x1.jpg", release_year: "1991", price: 600, genre: Genre.third, artist: Artist.find(9), slug: "low-end-theory" }])
```

Comment:
Can the genre & artist columns be set like that? Do you need to supply the id or does this automatically handle that?